ambari-agent/pom.xml

```diff
@@ -314,7 +314,7 @@
             <fileEncoding>utf-8</fileEncoding>
           </postremoveScriptlet>
 
-          <needarch>x86_64</needarch>
+          <needarch>aarch64</needarch>
```

Comment:
Can we keep **x86_64** as default one and make it dynamic..?

Reply:
Generate a build script build-ambari.sh to solve the problem. related link: https://github.com/apache/ambari/pull/3681/commits/6e8082a3a9c133ed40df603783113a9e8c83fce4